about/cascading-style-sheets.md

```diff
@@ -0,0 +1,74 @@
+# About Cascading Style Sheets
+
+## Properties
+
+#### display
+Usually block or inline, every element has a default display value.
+  * block - block-level elements starts on a new line and stretch left to right as far as they can. Example:
+    `<div>, <p>, <form>`
+  * inline - inline elements can wrap some text inside a paragraph without disrupting flow. Example:
+    `<span>, <a>`
+  * none - default value of JavaScript elements, used to hide and show elements without deleting and recreating them
+    * Different from visibility:hidden, which hides the element but still takes up the space it would if visible
```

Comment:
Probably want to cover more of these... Specifically, I'm looking for `inline` and `inline-block`...